ocs_ci/ocs/ui_selenium/const_xpath.py

```diff
@@ -0,0 +1,19 @@
+xpath_pvc_page = {
+    "Storage Tab": "/html/body/div[2]/div/div/div/div/div[1]/div/div/nav/ul/li[5]/button",
```

Comment:
Isn't there a better way than xpath? Small change in the page the test will stop working.
Like class="pf-c-nav__link" and its outerhtml "storage" word in it?
Or like you did int PVC page. Start querying from top html tag will stop working sooner or later.